cmd/kosli/assertArtifact.go

```diff
@@ -78,6 +79,7 @@ func newAssertArtifactCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&o.flowName, "flow", "f", "", flowNameFlag)
 	cmd.Flags().StringVar(&o.envName, "environment", "", envNameFlag)
 	cmd.Flags().StringVarP(&o.output, "output", "o", "table", outputFlag)
+	cmd.Flags().StringSliceVarP(&o.policyNames, "policy", "", []string{}, policyName)
```

Comment:
the "" is for the one character shortcut for the flag, since it is empty, we can use `StringSliceVar` instead of `StringSliceVarP` and get rid of that parameter